### DISCUSSION TITLE
How can I block port 3000 when you try to access it via browser http://public_ip:3000

### DISCUSSION THREAD

**Discussion by jrca025** [requarks/wiki#5018] [Help / Questions]
I am running wikijs with reverse proxy on apache with cloudflare argo tunnel so it will redirect to another domain. 

Although if you know the port that wikijs is using, it can still connect to it if you include the port number on the browser.

**Comment by NGPixel** [comment]
That's not really a Wiki.js issue... You should ask on stackoverflow or more network-related forums.
